src/View/SSViewer_BasicIteratorSupport.php

```diff
@@ -24,6 +26,8 @@ class SSViewer_BasicIteratorSupport implements TemplateIteratorProvider
     public static function get_template_iterator_variables()
     {
         return [
+            'isFirst',
+            'isLast',
```

Comment:
Let's use uppercase first letters, to retain the existing convention.

Reply:
Sure thing, happy to do so.